code/game/machinery/nuclearbomb.dm

```diff
@@ -413,6 +413,15 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
 				continue
 			alive_mobs |= current_mob
 
+	for(var/datum/interior/interior in SSinterior.interiors)
+		if(interior?.exterior?.z == z)
+			for(var/mob/living/passenger in interior.get_passengers())
+				if(!(passenger in (alive_mobs + dead_mobs)))
+					if(passenger.stat != DEAD)
+						passenger.death(create_cause_data("nuclear explosion"))
+					for(var/obj/item/alien_embryo/embryo in passenger)
+						qdel(embryo)
```

Comment:
```suggestion
		if(!interior.exterior || interior.exterior.z != z)
			continue
	
		for(var/mob/living/passenger in interior.get_passengers())
			if(!(passenger in (alive_mobs + dead_mobs)))
				if(passenger.stat != DEAD)
					passenger.death(create_cause_data("nuclear explosion"))
				for(var/obj/item/alien_embryo/embryo in passenger)
					qdel(embryo)
```

Reply:
functionally the same you just want your grubby hands on my commits grrrr :P